tests/test_device_validator.py

```diff
@@ -0,0 +1,185 @@
+"""
+Copyright 2018 Grid Singularity
+This file is part of D3A.
+
+This program is free software: you can redistribute it and/or modify
+it under the terms of the GNU General Public License as published by
+the Free Software Foundation, either version 3 of the License, or
+(at your option) any later version.
+
+This program is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+GNU General Public License for more details.
+
+You should have received a copy of the GNU General Public License
+along with this program.  If not, see <http://www.gnu.org/licenses/>.
+"""
+import unittest
+import json
+
+from d3a_interface.device_validator import validate_load_device, validate_pv_device, \
+    validate_storage_device
+from d3a_interface.constants_limits import ConstSettings
+
+GeneralSettings = ConstSettings.GeneralSettings
+LoadSettings = ConstSettings.LoadSettings
+PvSettings = ConstSettings.PVSettings
+StorageSettings = ConstSettings.StorageSettings
+
+
+class TestValidateDeviceSettings(unittest.TestCase):
+
+    def test_load_device_setting(self):
+        assert validate_load_device(avg_power_W=-1) == \
```

Comment:
If this was the reason for returning a boolean value instead of an exception, you can check whether an exception is raised with the context manager assertRaises: 
`with self.assertRaises(ValueError):
    validate_load_device(avg_power_W=-1)`